src/inference/src/ie_core.cpp

```diff
@@ -1161,7 +1161,7 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                 desc.pluginCreateFunc(plugin_impl);
                 plugin = InferencePlugin{plugin_impl, {}};
             } else {
-                so = ov::util::load_shared_object(desc.libraryLocation.c_str());
+                so = ov::util::load_shared_object_safely(desc.libraryLocation.c_str());
```

Comment:
why not to change behavior of `load_shared_object` to load libraries safe?

Reply:
I suppose unsafe code is also in `findPluginXML` where we try to find plugins.xml in different locations..

Reply:
> why not to change behavior of load_shared_object to load libraries safe?
It may be done, but case with extensions should support loading not only by absolute path

> I suppose unsafe code is also in findPluginXML where we try to find plugins.xml in different locations..
I tried to change logic of `getPluginPath` to prevent preparation of relative paths, but it causes exceptions on "read" stage and may be unuseful for cases when, for example, plugins.xml contains AUTO plugin record, but in real there is no AUTO plugin library. So exception should be thrown only on "load" stage.
So, I guess, the same is applicable for `findPluginXML()`